addons/website/views/snippets/s_tabs.xml

```diff
@@ -64,32 +64,53 @@
         </div>
         <div data-js="NavTabsStyle" data-selector="section" data-target=".s_tabs_main">
             <we-select string="Style">
-                <we-button data-set-style="tabs" data-name="tabs_opt" data-trigger="horizontal_opt">Tabs</we-button>
-                <we-button data-set-style="pills" data-name="pills_opt" data-trigger="tabs_color_opt" data-trigger-value="">Buttons</we-button>
+                <we-button data-set-style="nav-underline" data-name="underline_opt">Underline</we-button>
+                <we-button data-set-style="nav-tabs" data-name="tabs_opt" data-trigger="horizontal_opt">Tabs</we-button>
+                <we-button data-set-style="nav-buttons" data-name="buttons_opt" data-trigger="horizontal_opt">Buttons</we-button>
+                <we-button data-set-style="nav-pills" data-name="pills_opt">Pills</we-button>
             </we-select>
-            <we-colorpicker string="Tabs Color" data-dependencies="tabs_opt" data-name="tabs_color_opt" data-select-style="true" data-css-property="background-color" data-color-prefix="bg-"/>
-            <we-select string="Direction" data-dependencies="pills_opt">
+            <we-colorpicker
+                string="Header Background"
+                class="o_we_sublevel_1"
+                data-apply-to=".s_tabs_nav"
+                data-select-style="true"
+                data-css-property="--tabs-bg-color"
```

Comment:
> [..]  except that i didn't manage to retrieve the current colors applied in the colorpicker

Here you're instructing thr editor to edit `--tabs-bg-color`, but since this variable it's not set with a value by default,  the editor it's not able to retrieve a value to show you ;)

To solve the issue just define `--tabs-bg-color` (and `--tabs-link-color`) in the CSS and with a low level of specificity ... it will be overridden by the editor ;)